website/docs/getting-started/setup/upgrade-to-business-edition/docker.mdx

```diff
@@ -1,77 +1,52 @@
 ---
-description: Follow the steps to upgrade the Appsmith community docker installation to a Business Edition.
+description: Follow the steps to upgrade the Appsmith community Docker installation to a Business Edition.
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Docker
 
-Follow the steps listed below to upgrade your Appsmith Community Edition (CE) to a Appsmith Business Edition (BE) installed on docker.
+The page provides steps to upgrade the Appsmith Docker Community Edition to the Business Edition.
 
-## Generate license key
+## Prerequisites
 
-Sign up on [customer.appsmith.com](https://customer.appsmith.com/) and generate a trial license key.
+- Sign up on [customer.appsmith.com](https://customer.appsmith.com/) and generate a trial license key.
+- Ensure you have at least 2 GB of free storage space for running the backup process.
+- Create a backup of the Appsmith instance before upgrading. For more information, see [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#backup-instance)
+- Ensure that you have the latest Appsmith version deployed on your instance. To update Appsmith, see the [Update Appsmith](/getting-started/setup/instance-management/update-appsmith) section.
 
-## Backup data
+## Upgrade Appsmith instance
 
-The Community and Business editions use MongoDB, so you don't have to migrate the database. It's still recommended to [backup the Community Edition data](/getting-started/setup/instance-management/appsmithctl#backup). Ensure that you backup the database with the docker instance up and running.
+Follow these steps to upgrade your Appsmith instance:
 
-## Update image
-
-If you are using `docker run`, head to [Step 4: Recreate the instance](#recreate-the-instance).
-
-Go to the directory where the `docker-compose.yml` file is located. Open the `docker-compose.yml` file, and look for the `image:` key. Update the image in the `docker-compose.yml` file from Community Edition (_appsmith-ce_) to Business Edition (_appsmith-ee_). 
-
-The updated file should have the image attribute set as below and save the file:
+1. Go to the directory where the `docker-compose.yml` file is located.
+2. Open the `docker-compose.yml` file, and search for the `image:` key.
+3. Update the value of the `image:` key as below:
 
 ``` yaml
   services:
     appsmith:
     #highlight-next-line
       image: index.docker.io/appsmith/appsmith-ee
 ```
+4. Save the file
 
-## Recreate the instance
-
-<Tabs groupId="dockercommands" queryString="current-docker-command">
- <TabItem label="Docker Compose" value="using-docker-compose-command">
- You can recreate the instance by running the below command:
-
-```bash
-docker-compose up -d
-```
-
-:::note
-If you have **only** replaced the license key then you have to run the following command to restart the instance.
+5. Recreate the instance with:
 
 ```bash
-docker-compose restart appsmith
+docker-compose up -d --force-recreate
```

Comment:
This would work for the replace license situation as well, right?

Reply:
The license key is no longer needed to be updated in the `docker-compose.yml` file, so the above command should suffice. CC: @sum35h